k6/Dockerfile

```
@@ -0,0 +1,72 @@
+RUN npm install
+
+# Copy source files
+COPY . .
+
+# Build all bundles (conformance, library, demos)
+RUN pnpm run build:bundle
+
+# =============================================================================
+# Stage 3: Runtime image with custom K6
+# =============================================================================
+FROM alpine:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)